Src/WitsmlExplorer.Api/Services/WellService.cs

```diff
@@ -31,24 +33,25 @@ public WellService(IWitsmlClientProvider witsmlClientProvider, IWellboreService
             _wellboreService = wellboreService;
         }
 
-        public async Task<IEnumerable<Well>> GetWells()
+        public async Task<IList> GetWells()
```

Comment:
Looks like you accidentally lost `<Well>`

Reply:
yes, you are right, thanks. fixed.